src/snapshotter.js

```diff
@@ -29,6 +29,7 @@ export default class SnapShotter {
     },
     selenium,
     onComplete,
+    onInfo,
```

Comment:
does this need the readme updating?

Reply:
Its just internal so not needed to update ReadMe.(same as onError there is no mention in readme)